tedana/combine.py

```diff
@@ -107,8 +107,8 @@ def make_optcom(data, tes, mask, t2s=None, combmode='t2s', verbose=True):
         Concatenated BOLD data.
     tes : (E,) :obj:`numpy.ndarray`
         Array of TEs, in seconds.
-    mask : (S,) :obj:`numpy.ndarray`
-        Brain mask in 3D array.
+    adaptive_mask : (S,) :obj:`numpy.ndarray`
+        Brain adaptive_mask in 3D array.
```

Comment:
adaptive_mask is just the number of good echoes and, if the number is less than the total number of echoes, there is an assumption that the latest echoes will be the ones excluded from the mask. This seems like a reasonable assumption, but any function that applies the adaptive_mask should note this assumption in the documentation. I suspect this will prevent confusion & headaches in the future.

Reply:
Great point. I will expand the docstring.